.gitlab-ci.yml

```diff
@@ -44,25 +44,18 @@ deploy_qa:
     - pip install ecs-deploy==1.11.0
     - aws ssm get-parameters-by-path --region $AWS_DEFAULT_REGION --path /qa/alegre/ --recursive --with-decryption --output text --query "Parameters[].[Name]" | sed -E 's#/qa/alegre/##' > env.qa.names
     - for NAME in `cat env.qa.names`; do echo -n "-s qa-alegre-c $NAME /qa/alegre/$NAME " >> qa-alegre-c.env.args; done
-    - echo -n "-s qa-alegre-c GITHUB_TOKEN arn:aws:secretsmanager:eu-west-1:848416313321:secret:GithubToken-Plain-BUhwIw" >> qa-alegre-c.env.args
```

Comment:
I'm not familiar with this script, but it seems like it is pulling secrets into the ENV for the various processes.  With the GITHUB_TOKEN removal, is there an alternate way those environment variables are being set?

Reply:
Hello @skyemeedan ! We simply don't need this token any more. In the past, we configured applications by cloning a `configurator` repo with configs in it. This needed a GITHUB_TOKEN. However, in the sew SSM universe this is no longer necessary.